src/components/shared/LrSvgIcon/LrSvgIcon.vue

```diff
@@ -0,0 +1,69 @@
+<!--
+* CORTX-CSM: CORTX Management web.
+* Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as published
+* by the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Affero General Public License for more details.
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <https://www.gnu.org/licenses/>.
+* For any questions about this software or licensing,
+* please email opensource@seagate.com.
+-->
+<template>
+  <div class="icon-container action-btn" v-if="icon">
+    <img
+      :src="require(`@/assets/icons/${icon}`)"
+      @click="$emit('click')"
+      class="action-btn-block"
+      alt="logo"
+    />
+    <img
+      v-if="hoverIcon"
+      :src="require(`@/assets/icons/${hoverIcon}`)"
+      @click="$emit('click')"
+      class="action-btn-hover"
+      alt="logo"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop } from "vue-property-decorator";
+
+@Component({
+  name: "LrSvgIcon",
+  components: {},
+})
+export default class LrSvgIcon extends Vue {
+  @Prop({ required: true }) private icon: string;
+  @Prop({ required: false }) private hoverIcon: string;
```

Comment:
The prop is optional but it is required on line 27. If the hoverIcon prop is not passed, use the same icon instead.

Reply:
line no 26  v-if="hoverIcon" so we won't need hover effect if this icon is not provided

Reply:
Okay

Reply:
change added to handle icon disappear on hover